scripts/tests.e2e.persistent.sh

```diff
@@ -9,18 +9,15 @@ set -euo pipefail
 
 # e.g.,
 # ./scripts/build.sh
-# ./scripts/tests.e2e.persistent_network.sh ./build/avalanchego
+# ./scripts/tests.e2e.persistent.sh --ginkgo.label-filter=x               # All arguments are supplied to ginkgo
+# E2E_SERIAL=1 ./scripts/tests.e2e.sh                                     # Run tests serially
+# AVALANCHEGO_PATH=./build/avalanchego ./scripts/tests.e2e.persistent.sh  # Customization of avalanchego path
```

Comment:
:pray:

Reply:
can we also add 
```
E2E_USE_PERSISTENT_NETWORK=1
```
I get it seems self explanatory but all the options should be at the top for quick review IMO.